riscv-sbi.adoc

```diff
@@ -0,0 +1,174 @@
+# RISC-V Supervisor Binary Interface Specification
+
+## Copyright and license information
+
+This RISC-V SBI specification is
+
+[%hardbreaks]
+(C) 2019 Palmer Dabbelt <palmer@sifive.com>
+(C) 2019 Atish Parta <atish.parta@wdc.com>
```

Comment:
I am not aware of the procedures adopted in RISC-V community, so pardon my ignorance.

I wonder if having individuals as copyright holders is the right thing to do. I assume, in case of a copyright claim the Foundation would have to get approval from the Copyright holders in order to respond and represent them.

/cc @kasanovic @rickoco